tree_test.go

```diff
@@ -205,13 +204,14 @@ func TestTree_Build_Load(t *testing.T) {
 
 	// export the tree at version 12,000 and import it into a sql db
 	ctx := context.Background()
+	traverseOrder := PreOrder
```

Comment:
Can you add test for post-order traversal.

Reply:
Yea, let's have this test run for both pre and post order.